Examples/cChocoFeatureExample.ps1

```diff
@@ -0,0 +1,28 @@
+
+
+configuration ChocoFeatures {
+
+    Import-DscResource -ModuleName cChoco
+
+    Node 'localhost' {
+
+        cChocoFeature allowGlobalConfirmation {
+
+            FeatureName = "allowGlobalConfirmation"
```

Comment:
I've had a bit of a deep dive and asked around the UKPowerShell group, I don't think this functionality is supported in DSC at the moment. As the parameter functions as a key for the configuration, it would need to be realized at compile time so something like a default parameter would not work. 

Have you any other ideas? potentially a good feature request for the PowerShell team, I've duplicated resourceName / Key a number of times in real world use.